lib/elixir/src/elixir.erl

```diff
@@ -487,3 +488,6 @@ handle_parsing_opts(File, Opts) ->
   put(elixir_parser_columns, Columns),
   put(elixir_token_metadata, TokenMetadata),
   put(elixir_literal_encoder, LiteralEncoder).
+
+with_as_maybe() ->
+  lists:member(maybe_expr, erl_features:enabled()).
```

Comment:
We may need to check for the OTP release and only check for `erl_features:enabled()` if 26+ or later. There should be examples in this file of doing that!